user/zrepl-dsh2dsh/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "zrepl-dsh2dsh"
+pkgver = "0.9.2"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+depends = ["zfs"]
+provides = [self.with_pkgver("zrepl")]
+pkgdesc = "One-stop ZFS backup & replication solution - dsh2dsh enhanced fork"
+maintainer = "Robert David <robert.david@posteo.net>"
+license = "MIT"
+url = "https://github.com/dsh2dsh/zrepl"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "1262c854c32e66cf67dd1d8c2ca6e546b6d42100c9bca9857ba37b6a16a5b1d1"
+
+# check needs to run zfs command
+options = ["!check"]
+
+def install(self):
+    self.install_bin("build/zrepl")
+    self.install_file("dist/freebsd/etc/zrepl/zrepl.yml", "etc/zrepl", name="zrepl.yml.sample")
```

Comment:
examples don't go in `/etc`

Reply:
I may also change it to be zrepl.yml as an default config. But I feel it is more like a sample than default config to be used as is.